expressEngine/orderItems/controller/index.js

```diff
@@ -0,0 +1,5 @@
+import OrderItemsController from './orderItems';
+
+export {
+  OrderItemsController,
```

Comment:
Single export file should be exported as default.

Reply:
At the initial stage, I wanted to keep it optional